### ISSUE TITLE
Add ability specifify a anchor tag when linking somewhere

### ISSUE THREAD

**Issue by codyjames** [fruitstudios/craft-linkit#39]
Sometimes you want to link to a specific section of a page/entry/etc.

**Comment by proimage** [comment]
At minimum, this would be doable if an exception were added to the URL validation that didn't require a proper URL formatting if the string begins with a #.

**Comment by codyjames** [comment]
I think it would be nice if it could be an additional field. So you can still link to an entry but also to an anchor on the page of said entry.

**Comment by samhibberd** [comment]
Thanks @codyjames Added support for this in [1.1.5](https://github.com/fruitstudios/craft-linkit/releases/tag/1.1.5)

We decided to run with @proimage suggestion, unless people think there is a real use case for an Anchor link type? We feel it might be adding more layers of settings. Happy to reconsider though.

**Comment by jsunsawyer** [comment]
@samhibberd I don't think that we need a new `Anchor` link type, but it would be nice to still be able to link to an anchor for a given entry or category. This ensures that URLs stay updated, handles translated versions for multi-sites, etc.

Just an optional anchor field that could be added would be great.

![screen shot 2019-01-14 at 10 53 12 am](https://user-images.githubusercontent.com/1007191/51133903-f8795c80-17ea-11e9-8723-0ea4394d21ce.png)

If the anchor field is populated, it gets appended to the entry or category url.

**Comment by jsunsawyer** [comment]
Also, it might be worth considering query strings and coming up with a single solution for both.

**Comment by samhibberd** [comment]
@jsunsawyer this could work nicely, we can look at this after we have got 3.1 support released.
